src/globus_sdk/experimental/auth_requirements_error/_auth_requirements_error.py

```diff
@@ -68,7 +78,10 @@ def __init__(
         self.required_scopes = _validators.opt_str_list(
             "required_scopes", required_scopes
         )
-        self.prompt = _validators.opt_str("prompt", prompt)
+        if prompt in [None, "login"]:
+            self.prompt = prompt
+        else:
+            raise _validators.ValidationError("'prompt' must be 'login' or null")
```

Comment:
I kinda assumed that this was in relation to json parsing because it used null instead of None, but I'm not sure I completely understand how this object is meant to be used (which is part of why I'm hesitant to mess around with it)